lib/web_console/middleware.rb

```diff
@@ -103,7 +103,7 @@ def id_for_repl_session_stack_frame_change(request)
 
       def update_repl_session(id, request)
         json_response_with_session(id, request) do |session|
-          { output: session.eval(request.params[:input]) }
+          { output: session.eval(request.params[:input], request.params[:rawdata] == 'true') }
```

Comment:
Maybe we can always include the variables in the JSON response, instead of relying on this conditional query parameter being set. This will also open it up for choosing the proper variable by the prefix on the clientside, so we can drop the WebAPI server module.